packages/blockchain-wallet-v4-frontend/src/config/index.js

```diff
@@ -1,13 +1,15 @@
 import Bitcoin from 'bitcoinjs-lib'
 import BitcoinCash from 'bitcoinforksjs-lib'
 
+const isTestnet = process.NODE_ENV === 'testnet'
+
 export default {
   WALLET_DATA_PATH: 'wallet.data',
   WALLET_PAYLOAD_PATH: 'wallet.payload',
   WALLET_SETTINGS_PATH: 'wallet.settings',
   WALLET_OPTIONS_PATH: 'wallet.options',
   WALLET_KVSTORE_PATH: 'wallet.kvstore',
-  NETWORK: Bitcoin.networks.bitcoin,
+  NETWORK_BITCOIN: Bitcoin.networks[isTestnet ? 'testnet' : 'bitcoin'],
   NETWORK_ETHEREUM: 1,
-  NETWORK_BCH: BitcoinCash.networks.bitcoin
+  NETWORK_BCH: BitcoinCash.networks[isTestnet ? 'testnet' : 'bitcoin']
```

Comment:
Small nitpick. We're trying to use currency codes instead of currency names. Can you change to NETWORK_BTC?

Reply:
If you hardcode `testnet` for now, I noticed a failure and helpful :) note here: http://github.com/blockchain/blockchain-wallet-v4-frontend/blob/e6b67ab820bb6e262dfb2f89ac3f3389d1c1fcb6/packages/blockchain-wallet-v4/src/types/HDAccount.js#L93

Reply:
I also noticed we are hardcoding `Bitcoin.networks.bitcoin` in a few places, which will need to be replaced by settings config

Reply:
@plondon Thank you! 

1. I will address the naming.
2. Good catch! I caught that TODO as well and has addressed it in local. 
3. Great! I will search for these hardcoding reference